### ISSUE TITLE
Play melodies with the dshot beacon

### ISSUE THREAD

**Issue by mathiasvr** [mathiasvr/bluejay#19]
We can now change the startup melody, but we also want to play melodies with the beacon signal.

**Comment by Quick-Flash** [comment]
Yes that would be fun to allow the dshot beacons to just turn on esc music rather than one tone of beeps. Would be fun to be searching for your drone only to have it rick rolling you.

**Comment by koen352003** [comment]
> We can now change the startup melody, but we also want to play melodies with the beacon signal.

I think I might have missed something, are you actually saying that the custom startup melody is already working? Didn't notice it being in the changelog, but if it's true than that's really quite amazing! I really like the work you and the others are doing around here, it's wonderfull to see this amount of support for the blheli_s esc's!

**Comment by mathiasvr** [comment]
@koen352003 Yes might be a bit overstated.

Version [0.12](https://github.com/mathiasvr/bluejay/releases/tag/v0.12-test) is currently in a test phase. You can try it with the newest version of bluejay configurator and soon esc configurator too. It isn't perfect yet but can play some simple rtttl melodies.

**Comment by howels** [comment]
Is it possible to have a beacon melody in addition to the startup melody? I urgently need to be able to rick-roll my fellow flyers.
